forc-plugins/forc-doc/src/cli.rs

```diff
@@ -22,8 +22,8 @@ forc_util::cli_examples! {
 pub struct Command {
     /// Path to the Forc.toml file. By default, forc-doc searches for the Forc.toml
     /// file in the current directory or any parent directory.
-    #[clap(long)]
-    pub manifest_path: Option<String>,
+    #[clap(long, alias = "manifest-path")]
```

Comment:
To be aligned with the `forc build` and `forc fmt`, we should also have the short `-p` argument here: `#[clap(short, long)]`.